app/assets/stylesheets/course/enrol_requests.css.scss

```diff
@@ -0,0 +1,43 @@
+#enrol_request_form {
+  min-width: 190px;
+  width: 190px;
+  a {
+    width: 100%;
+    margin: 5px 0px 5px 0px;
+  }
+}
+
+.bold-text {
+  font-weight: bold;
+}
+
+.enrol-request-row {
+  td {
+    vertical-align: middle !important;
+  }
+  .request-approve {
+    margin-right: 5px;
+  }
+  .request-approve, .request-delete {
+    padding-top: 2px;
+    padding-bottom: 2px;
+  }
+}
+
+.batch-controls {
+  input[type="submit"] {
+    /* Remove all decorations to look like normal text */
+    background: none;
+    border: none;
+    display: inline;
+    font: inherit;
+    margin: 0;
+    padding: 0;
+    outline: none;
+    outline-offset: 0;
+    /* Additional styles to look like a link */
+    color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.